packages/connect-query-core/proto/bigint.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+service BigIntService {
```

Comment:
Duplicating the Protobuf files will make this more difficult to maintain. The files will eventually drift, and we'll have surprising behavior. 

Do we still have duplicates if we only keep the files in each package that are required? If yes, we can move the files to a separate directory that's used by both packages (probably needs an update to turbo.json to make sure we re-generate when the files change). It's fine to generate to multiple places.

Alternatively, it can be an option to keep the files in a private package that provides generated code.

Reply:
We can simplify these I think. There's no point duplicating logic and tests between `connect-query` and `connect-query-core`.